.github/workflows/pkg.pr.new.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Any Commit
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '!**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [20]
```

Comment:
Should we use `lts/*`? And also we could directly specify the version below and don't need a matrix I think